src/renderer/component/categoryList/view.jsx

```diff
@@ -189,8 +200,17 @@ class CategoryList extends React.PureComponent<Props, State> {
   }
 
   render() {
-    const { category, names, categoryLink } = this.props;
+    const { category, categoryLink, names, claimsByChannel, claimsById } = this.props;
     const { canScrollNext, canScrollPrevious } = this.state;
+    const channel =
```

Comment:
you should also be able to use `makeSelectClaimsInChannelForCurrentPage`, which is what the channel page uses. Just get the first page of content and you won't have to worry about this logic.

Reply:
I have tried using in the [makeSelectClaimsInChannelForCurrentPage](https://github.com/lbryio/lbry-redux/blob/master/src/redux/selectors/claims.js#L73) selector in place of the other two and the logic that goes with them, and I keep running into problems. I'm passing `props.categoryLink` as the `uri`, which I think should be fine based on console logging props. I have tried various forms of the makeSelect pattern I found in the [RewardLink container-component](https://github.com/lbryio/lbry-app/blob/master/src/renderer/component/rewardLink/index.js#L11) to no avail, including rewriting the selectors, but there are so many not-working version I'm not sure what to push/show you.

Reply:
@skhameneh